enterprise/server/remote_execution/containers/ociruntime/ociruntime_test.go

```diff
@@ -34,6 +41,14 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 	"github.com/buildbuddy-io/buildbuddy/server/util/testing/flags"
 	"github.com/buildbuddy-io/buildbuddy/server/util/uuid"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/registry"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
nit: instead of aliasing to `v1` I think an alias of `containerregistry` or something shorter like `registry` would be more clear.

Reply:
renamed